Model GAP_6, Test surface-energy-111-unrelaxedsurface-energy-111-unrelaxed
Test run at 2018-10-04 22:50

model file: ../../../models/W/GAP_6/model.py
============================================================
import os

from quippy import Potential
import __builtin__

orig_dir = os.getcwd()
model_dir = os.path.dirname(__file__)
if model_dir != '':
    os.chdir(model_dir)

if os.path.exists('gp_iter6_sparse9k.xml.sparseX.GAP_2017_6_17_60_4_3_56_1651.bz2'):
    os.system('bunzip2 gp_iter6_sparse9k.xml.sparseX.GAP_2017_6_17_60_4_3_56_1651.bz2')

try:
    if hasattr(__builtin__, 'mpi_glob'):
        calculator = Potential(init_args='Potential xml_label="GAP_2013_6_24_60_12_58_8_327"',
                                               param_filename='gp.xml', mpi_obj=mpi_glob)
    else:
        calculator = Potential(init_args='Potential xml_label="GAP_2013_6_24_60_12_58_8_327"',
                                               param_filename='gp.xml')
    Potential.__str__ = lambda self: '<GAP Potential>'
finally:
    os.chdir(orig_dir)

no_checkpoint = True

name = 'GAP'
============================================================
test file: ../../../tests/W/surface-energy-111-unrelaxedsurface-energy-111-unrelaxed/test.py
============================================================
Traceback (most recent call last):
  File "run-model-test-cas.py", line 88, in <module>
    sys.stdout.write(open(test_file).read())
IOError: [Errno 2] No such file or directory: '../../../tests/W/surface-energy-111-unrelaxedsurface-energy-111-unrelaxed/test.py'
